decidim-admin/app/views/decidim/admin/features/_feature.html.erb

```diff
@@ -9,37 +9,25 @@
   </td>
   <td class="table-list__actions">
     <% if feature.manifest.admin_engine %>
-      <%= link_to manage_feature_path(participatory_process, feature), data: { tooltip: true, disable_hover: false }, title: t("actions.manage", scope: "decidim.admin"), class: "action-icon action-icon--manage" do %>
-        <%= icon "pencil" %>
-      <% end %>
+      <%= icon_link_to "pencil", manage_feature_path(participatory_process, feature), t("actions.manage", scope: "decidim.admin"), "action-icon--manage" %>
```

Comment:
You're using the argument `klass` in the wrong way, as this would actually generate `action-iconaction-icon--manage`. Same applies for the rest of usage.